src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -412,24 +412,19 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
         }
     }
 
-    @computed get isQueryValid() {
-        return this.resultsViewPageStore.hugoGeneSymbols.length <= AppConfig.serverConfig.query_gene_limit;
-    }
-
     @computed get pageContent(){
-        // if qeury invalid, return error page
-        if (!this.isQueryValid) {
-            return (
-                <div className="alert alert-warning queryInvalid" role="alert">
-                    <p>
-                        Queries are limited to 100 genes. Please <a href={`mailto:${AppConfig.serverConfig.skin_email_contact}`}>let us know</a> your use case(s) if you need to query more than 100 genes.
-                    </p>
-                </div>
-            );
-        }
-
         // if studies are complete but we don't have a tab id in route, we need to derive default
         return (<div>
+            {
+                // if qeury invalid, return error page
+                (this.resultsViewPageStore.isQueryInvalid) && (
```

Comment:
we should probably have a mechanism different types of invalidity (of which gene length is only one) and messaging for each one.  but since right now we only do gene length, this is ok.  maybe put a comment explaining that.